src/Rules/PhpDoc/VarTagTypeRuleHelper.php

```diff
@@ -127,6 +127,10 @@ private function containsPhpStanType(Type $type): bool
 
 	private function shouldVarTagTypeBeReported(Node\Expr $expr, Type $type, Type $varTagType): bool
 	{
+		if ($expr instanceof Expr\Array_ && $expr->items === []) {
+			return false;
```

Comment:
This is wrong. It's gonna silence this case:

```php
/** @var int */
$a = []; // nonsense
```

Reply:
Just fixed that.